test/index.js

```diff
@@ -813,3 +816,9 @@ test('contramap', function() {
 
   eq(Z.contramap(length, inc)('abc'), 4);
 });
+
+test('errors', function() {
+  throws(function() { Z.chainRec(Maybe); },
+         TypeError,
+         'ChainRec.methods.chainRec(...) is not a function');
+});
```

Comment:
Shall I remove the `throws` implementation and other changes related to that test then?